src/main/java/com/tailf/jnc/Utils.java

```diff
@@ -174,14 +174,14 @@ public static String wsCollapse(String value) {
         final String res = value.replaceAll(" +", " ");
 
         // Remove any leading and/or trailing space
-        final int startOffset = res.startsWith(" ") ? 1 : 0;
+        final int startOffset = res.charAt(0) == ' ' ? 1 : 0;
         final int stopOffset = res.length() > 1 && res.endsWith(" ") ? -1 : 0;
         return res.substring(startOffset, res.length() + stopOffset);
     }
```

Comment:
I'm no Java programmer but is there no trim function/method?

Reply:
Definitely, my aim was to do none/least required changes to code unrelated to lint resolving, to prevent potential bug introductions. 
There is definitely lots of places where code can be simplified or enhanced, but i didn't want to risk it in the big batch de-linting PR.